Regenerate code for 849

valassi opened the pull request:
Hi @oliviermattelaer this is just regenerating code for your #849.
Ok for me (as long as you want 849 - I trust you on that)
Can you please review? Thanks
Andrea

oliviermattelaer reviewed (approved):
Yes, this is good to go for me.

Thanks,

Olivier

valassi commented:
> Yes, this is good to go for me.
> 
> Thanks,
> 
> Olivier

Thanks Olivier :-)

I had forgotten that 849 has conflicts, fixed here. So I changed this to be merged into less_patch for 849, then I will merge 849 into master.

Merging now.
